src/box/xrow.c

```diff
@@ -1522,6 +1529,13 @@ xrow_decode_raft_local(const struct xrow_header *row, struct raft_request *r,
 	return xrow_decode_raft_impl(row, r, vclock, GROUP_LOCAL);
 }
 
+int
+xrow_decode_raft(const struct xrow_header *row, struct raft_request *r,
+		 struct vclock *vclock)
+{
+	return xrow_decode_raft_impl(row, r, vclock, GROUP_DEFAULT);
+}
```

Comment:
> I would suggest to have just one xrow_decode_raft which can work with any group (which won't even check it). That would be a bit simpler.

I intentionally did that in order to fortify the code execution. I suppose, this way the code will be more strict and we'll think twice, whether the row, we're reading, is local or remote, since RAFT can be both. This way we won't repeat this mistake in the future.

I'd rather leave as it is. Let's see what @sergepetrenko think here

Reply:
I think you could put the `group_id` into `raft_request`/`raft_msg` structs and make a single version of `xrow_encode/decode_raft` which handles the group id.

That's because we already have `raft_checkpoin_local` and `raft_checkpoint_remote`, which may set group id as well.

Reply:
> I think you could put the group_id into raft_request/raft_msg structs and make a single version of xrow_encode/decode_raft which handles the group id.

Great idea! Done that. So much cleaner now